package.json

```diff
@@ -28,7 +28,7 @@
     "object.assign": "^4.0.1",
     "safe-json-parse": "^4.0.0",
     "tsml": "1.0.1",
-    "videojs-font": "1.3.0",
+    "videojs-font": "git+https://github.com/videojs/font.git#182aa4c2ee46cca7ca00155c4cdbd89acbad5a65",
```

Comment:
I released 1.4.0 on npm. Check it out. https://github.com/videojs/font/commit/57161e1988df1b4ac86d527b59382db08050b2d9
If it works for you, I'll release it to the CDN tomorrow.